### PR TITLE
[Event Hubs] Processor Load Balancing Defaults

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The focus of these changes is to apply learnings from profiling and testing of real-world application scenarios to the default values that control load balancing behavior for the processor in order to increase stability and performance.

### CODE REVIEW

File: sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor.cs [hunk 1]
```diff
@@ -1455,10 +1455,12 @@ private async Task RunProcessingAsync(CancellationToken cancellationToken)
 
             try
             {
+                ValueStopwatch cycleDuration;
+
                 var connection = CreateConnection();
                 await using var connectionAwaiter = connection.ConfigureAwait(false);
 
-                ValueStopwatch cycleDuration;
+                //var firstIteration = true;
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor.cs:1463]: Remove;  this was debugging code not intended to be included.

**Review by christothes — APPROVED** [review]

**Comment by jsquire** [comment]
/ azp run net - eventhub - tests

**Pull request merged**